apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -20,6 +20,7 @@ type AvailableTimesProps = {
   afterBufferTime: number;
   eventTypeId: number;
   eventLength: number;
+  recurringCount: number | undefined;
```

Comment:
This is to keep track of the selected number of occurrences the attendee choses, less or equal to the Max number of occurrences the organizer set up in the event type configuration